exampleCourse/questions/demo/workspace/rstudio/question.html

```diff
@@ -1,6 +1,25 @@
 <pl-question-panel>
+
 <markdown>
+----
+
```

Comment:
Nit for consistency with other question:

```suggestion
```

Reply:
Done